.ci/Makefile

```diff
@@ -35,14 +35,6 @@ clean:
 env:
 	@env | sort
 
-# Create Docker image
```

Comment:
nit: these lines above could now be removed:

```
DIST_DIR = /build/dist/nodejs
DOCKER_IMAGE_NAME = observability/apm-agent-nodejs
DOCKER_REGISTRY = docker.elastic.co
```

Reply:
see https://github.com/elastic/apm-agent-nodejs/pull/3938/commits/a8430f4bdb72b9abdf8dbb28699689d353b6b3b7